### ISSUE TITLE
Where should patter cuesheets go?

### ISSUE THREAD

**Issue by sharmanr** [mpogue2/SquareDesk#772]
If I have a patter song and click the Patter tab if gives me a starting Sequence Sheet.  If I modify it then do a 'Save as" the defaulted directory is Lyrics.   If I save it there,  the next time I select that song it won't be found because for a patter song the lyrics directory is specifically excluded.
Where should it be saved?  We could save it in the patter directory but that doesn't seem right.  Should there be a specific directory for patter sequences (created if necessary) and make the default when saving a patter song?

**Comment by mpogue2** [comment]
Yeah, "lyrics" could have been named "cuesheets" or "figures" or something like that, and then used by both Patter and Singer.  But, I will also say, we have ZERO users of patter cuesheets at this point, AFAIK. :-)

So, I'm thinking that maybe we should NOT fix this (at least in the "cuesheet" way), and instead use aim at using SD as the sequences/figures repository for patter.  It's just about in barely-usable shape, at this point, I think.

Also, see #646 -- I really could use the ability during the patter to "peek ahead" to the figures I have setup in the singing call, so that I can run through that figure in the patter.  That might be the primary use case for the Lyrics tab (whose name changes to Patter right now -- we could change that so that it's always Lyrics)...

What do you think?

**Comment by sharmanr** [comment]
I only found this because I was testing the swapping heads and sides, not while actually using SquareDesk.  If new users tried to save sequences here it could be a problem. Was the reason for filtering out the searching in lyrics for a patter song to save time [when loading a patter song]?   Because removing that restriction would probably cause the tab to work the same for singers and patters.  Otherwise,  just remember to save in patter not lyrics for example and it works.
I haven't figured out how to use the SD stuff yet.
I like the idea of #646;  I think that sounds very useful.

**Comment by mpogue2** [comment]
@sharmanr I think the original reasons were:
- no need to search through lyrics at all, when we know that lyrics would not be relevant during patter.  Yes, this saves time, but more importantly, it avoids the "false positive" problem, where the title of some singing call kinda matches the name of the patter record, and it therefore shows up in the list (and it's useless).  
- the ability to attach SOME cuesheets to patter would be "poor-man's sequence feature", that is, we'd be able to write some sequences, attach them to a patter, and then use that to call those sequences during the patter.  This feature turned out to be completely unused, since all of the callers using SquareDesk are using other methods for looking at sequences (cards, iPad, textedit window, or the Tip Planner app that I subsequently wrote).
- Having the ability to attach cuesheets to patter would indeed make the tab work the same for singers and patters.

But, the cognitive load to stick patter cuesheets somewhere else, along with the fact that nobody used this feature, PLUS the new SD tab (which is designed like CSDS, for reading cards during patter), suggests to me that this feature is obsolete, and can be removed (perhaps by #646).
